### PR TITLE
teach node.js resolve tsx

### PR DESCRIPTION
**Pull request by tuoxiansp**

### CODE REVIEW

File: cli/index.js [hunk 1]
```diff
@@ -54,7 +55,13 @@ if (commands.includes(command)) {
             'ts-node/register',
             require.resolve(`./${command}.ts`),
         ],
-        { stdio: 'inherit' }
+        {
+            stdio: 'inherit',
+            env: {
+                ...process.env,
+                TS_NODE_PROJECT: path.join(__dirname, './tsconfig.json'),
```
Comment (rostislavbobo) [inline cli/index.js:62]: you don't need to provide the current directory (`./`) path
```js
TS_NODE_PROJECT: path.join(__dirname, 'tsconfig.json')
```

Comment (tuoxiansp) [reply cli/index.js]: Thanks a lot! Fixed.

**Comment by tuoxiansp** [comment]
Haven't found a good solution yet. 

https://github.com/microsoft/TypeScript/issues/25271#issuecomment-400868436

**Comment by tuoxiansp** [comment]
Seems if tsconfig is not a JSON file but some other dsl allow  simply extendable express can be good for us.

**Pull request merged**